apply/cloud_applier.go

```diff
@@ -132,9 +132,19 @@ func (c *CloudApplier) Apply() error {
 	}
 
 	cluster.Spec.Provider = common.BAREMETAL
-	err = utils.MarshalYamlToFile(common.TmpClusterfile, cluster)
-	if err != nil {
-		return fmt.Errorf("marshal tmp cluster file failed %v", err)
+	clusterFile := cluster.GetAnnotationsByKey(common.ClusterfileName)
```

Comment:
I think we should generate the tmpClusterfile which append Config and Plugin.

1. generateTmpClusterfile()
       1. dump v1.Cluster
       2. append v1.Config
       3. append v1.Plugin
2. sendTmpClusterfile()